test/apiv2/45-system.at

```diff
@@ -25,6 +25,26 @@ t GET system/df 200 '.Volumes[0].Name=foo1'
 
 t GET libpod/system/df 200 '.Volumes[0].VolumeName=foo1'
 
+# Verify that no containers reference the volume
+t GET system/df 200 '.Volumes[0].RefCount=0'
+t GET libpod/system/df 200 '.Volumes[0].RefCount=0'
```

Comment:
Looks unimplemented: I see no field with anything looking like `RefCount`.

Handy tip: this is how I debug APIv2 tests:
```bash
t GET ... 200 sdf   <------ the "sdf" is the important thing
```
Since `sdf` does not match, the test just dumps the entire return value, in this case:
```
not ok 21 [45-system] GET libpod/system/df : output                                                                                                             
#  expected: dsf                                                                                                                                                
#    actual: {"Images":[],"Containers":[],"Volumes":[{"VolumeName":"foo1","Links":0,"Size":0,"ReclaimableSize":0}]}                                             
not ok 22 [45-system] GET libpod/system/df : .Volumes[0].RefCount                                                                                               
#  expected: 0                                                                                                                                                  
#    actual: null              
```
And OBTW the `POST containers/create` is failing too